cartographer_ros/cartographer_ros/node.cc

```diff
@@ -202,7 +202,13 @@ void Node::PublishTrajectoryStates(const ::ros::WallTimerEvent& timer_event) {
     const ::cartographer::common::Time now = std::max(
         FromRos(ros::Time::now()), extrapolator.GetLastExtrapolatedTime());
     stamped_transform.header.stamp = ToRos(now);
-    const Rigid3d tracking_to_local = extrapolator.ExtrapolatePose(now);
+
```

Comment:
readability: initialize non-const as it used to be.
After that, 
if (options.project) { tracking_to_local = Embed3D(tracking_to_local); }